src/context/index.js

```diff
@@ -2,7 +2,9 @@
 
 import AuthProvider from "./AuthProvider";
 import VideoProvider from "./VideoProvider";
+import LikeProvider from "./LikeProvider";
```

Comment:
create an index.js for all the providers and then import from that index.js

Reply:
Didn't get you, This is same index file you are talking about I guess. I have imported all the contexts and then using ir from same index file